graphql_compiler/tests/test_schema.py

```diff
@@ -26,11 +27,26 @@ def _get_directives_in_string_form(directives):
             )
             fake_schema = GraphQLSchema(fake_query_type, directives=directives)
 
-            schema_lines = [line.strip() for line in print_schema(fake_schema).split("\n")]
+            # Split schema on double line breaks where the following character is not a space.
+            # It is not possible to simply split on double line breaks because print_schema puts a
+            # double line break between GraphQLArguments. The not space character is retained and
+            # reattached to the rest of the line.
+            split_schema_lines = [
+                line.strip() for line in re.split("\n\n([^ ])", print_schema(fake_schema))
+            ]
+
+            # Reattach the deliminator's character to the rest of the line. The first line does
```

Comment:
nit: I think `delimiter` might be the more commonly-recognized nomenclature here and in the variable name below